### ISSUE TITLE
Really tragic video quality between two iphones

### ISSUE THREAD

**Issue by msavara** [signalapp/Signal-iOS#5294]
Hi, do you have a feeling that Signal is starting to have very significantly bad video quality of video calls? I've been noticing it for a while and it's just really probably the worst on the market right now. When I play a telegram video or FaceTime video in the same place at the same time, the quality is significant better. I videocall through the signal to the same person and it's really a tragedy. Don't you have a similar feeling? (I tested it between two iphones 12 Pro)

And please, go to hell with something "there is missing info. " I tried to do that a long time ago. But nobody was interested and it was closed... but the bug is still there and nobody cares.  [My old bugreport](https://github.com/signalapp/Signal-iOS/issues/4484)

**Comment by msavara** [comment]
YES IT'S STILL RELEVANT!

**Comment by msavara** [comment]
> Is this still relevant? If so, what is blocking it? Is there anything you can do to help move it forward?
> 
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

YES IT'S STILL RELEVANT!

**Comment by alexbhandari** [comment]
Agreed this seems to still be an issue. I am also having issues with video quality coming from iPhones in signal today. Quality in general I can see is worse than in whatsapp. Additionally I have observed that when there is motion in the video the frame rate is extremely slow and quality is very bad.

**Comment by alexbhandari** [comment]
More people having this issue: https://www.reddit.com/r/signal/comments/vk94pp/video_call_lagging_iphone_to_android/?utm_medium=android_app&utm_source=share

**Comment by msavara** [comment]
Yes, I know. I'm incredibly impressed by this inactivity of Signal. There have been several bug reports and some of them are really old. Nobody has reacted at all from owners. This is absolutely unbelievable. 
There is also perfect answers on Reddit "Make a debug log and report the issue to Signal devs. " there was tons and tons of bug reports and nobody cares.

**Comment by edsimpsons83** [comment]
Agreed, suprised this isn’t a larger priority! Disappoint to see it not even acknowledged by the developers.

**Comment by johnwittacre** [comment]
The top post on the signal sub reddit last week is everyone complaint about video quality. Would  be great to know if this is going to be fixed. 

https://old.reddit.com/r/signal/comments/x2k8r5/why_do_i_need_to_use_whatsapp_still/

**Comment by wtangofoxtrot** [comment]
Relying on facetime when possible now for calling, signal over a local wifi connection has much worse call quality than facetime has when calling the other side of the world. I think blackberry messenger 10 years ago had better quality video for group and regular calls.

**Comment by msavara** [comment]
Still relevant!

**Comment by clemlesne** [comment]
I confirm this is still relevant. The call quality is miserable.

**Comment by berlincount** [comment]
Having the same problem.

**Comment by scueZ** [comment]
Any update on this issue?

**Comment by johnwittacre** [comment]
Still bad video quality on ios for me.

**Comment by Miguel-Signal** [comment]
@scueZ @johnwittacre 

Could you please recreate this issue then write into support@signal.com with debug logs from both your device and debug logs from the recipient of the call?

To share a debug log from Signal iOS, tap your profile, Settings, Help, Submit Debug Log, Copy Link.

**Comment by scueZ** [comment]
Lately i have realised that the video call quality hasn't been bad in comparison to what i have seen earlier, and to benchmark i tried comparing it with telegeram/facetime. I will see if i face the issue again and will send the logs as mentioned.

**Comment by einarpersson** [comment]
My experience is that video calls on Signal has a sub-par quality (compared to similar apps). I'm trying to get friends and family to move to Signal (since it is a not-for-profit) but this is definitely something that makes it harder.

I am using Android (Samsung S21) and I have a great connection.

**Comment by DavidPopesc** [comment]
Somewhat unrelated, can we have signal video over LAN, similar to FaceTime? 

Also, a potential issue could be that Carriers might not be prioritizing Signal data like they are with FaceTime and other video apps. Signal might not be grouped with the video call QCI.
